rules/php74/src/Rector/LNumber/AddLiteralSeparatorToNumberRector.php

```diff
@@ -96,6 +96,7 @@ public function refactor(Node $node): ?Node
         }
 
         $node->value = $literalSeparatedNumber;
+        //              ^^^ When becomes a string, classes that expect LNumber to have int get broken.
```

Comment:
The reason for the problem.
`LNumber` can not be a string
https://github.com/nikic/PHP-Parser/blob/master/lib/PhpParser/Node/Scalar/LNumber.php#L25

Reply:
Explains why different Rectores failed in stack trace when I collected a minimum reproduction set